hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -1191,6 +1194,26 @@ public void testIfWeCarryRecordVersionInfoToReplicas() {
     }
 
     @Test
+    public void github_11489() throws Exception {
```

Comment:
This is a bad name, since it doesn't describe what it does. You can put a link to the issue in JavaDoc, e.g.
```java
    /**
     * Test for https://github.com/hazelcast/hazelcast/issues/11489
     */
```